core_impl/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -221,6 +225,16 @@ Timestamp getStart() {
   }
 
   /**
+   * Returns the milliseconds representation of this {@code Duration}.
+   *
+   * @param duration {@code Duration}.
+   * @return the milliseconds representation of a {@code Duration}.
+   */
+  static long toMillis(Duration duration) {
```

Comment:
This is package-private rather than private for testing, correct? So two questions:
1. In open source, do we annotate with @VisibleForTesting or mark this in any way?
2. Should the comments be javadoc? I seem to recall that we were only going to add javadoc for public things but I don't recall if that changed.

Reply:
Something like this is clearly a utility that should probably be in a util class - that may not be true for all package private methods but agree, we can look into that in a subsequent pr.